invenio_orcid/config.py

```diff
@@ -26,6 +26,12 @@
 
 ORCID_SYNCHRONIZATION_ENABLED = True
 
+ORCID_AUTHORS_SEARCH_CLASS = "current_app.config[RECORDS_REST_ENDPOINTS]" \
```

Comment:
You can do it as default when set to `None`, but not in this way. Preferably, just specify the importable string directly here.